docs/pages/app-signing/apple-team-permissions.mdx

```diff
@@ -0,0 +1,48 @@
+---
+title: Apple Developer account permissions for EAS Build
+sidebar_title: Apple Developer account permissions
+description: Learn about the Apple Developer account membership requirements for creating an EAS Build.
+---
+
+import { Terminal } from '~/ui/components/Snippet';
+import { Step } from '~/ui/components/Step';
+
+When creating an EAS Build for iOS, an Apple Developer account is required. This account has necessary permissions to submit the app for review, manage app's distribution, and generate required [app signing credentials](/app-signing/managed-credentials/#generating-app-signing-credentials) such as certificates, identifiers, and profiles.
```

Comment:
```suggestion
When creating a build for iOS Device on EAS, an Apple Developer account is required. This account has necessary permissions to submit the app for review, manage app's distribution, and generate required [app signing credentials](/app-signing/managed-credentials/#generating-app-signing-credentials) such as certificates, identifiers, and profiles.
```
^^ you probably have a better idea of how to word it, but the idea is, simulator builds don't require an Apple Developer account. This is specific for build that can run on physical iPhones.

Reply:
That's true! good catch!